resources/Materials/Examples/UsdPreviewSurface/usd_preview_normal_map.mtlx

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
```

Comment:
Instead of creating a normal-map-only example for UsdPreviewSurface, it might be more compelling to convert our existing procedural brick example from Standard Surface to UsdPreviewSurface.  Are there additional hurdles that we'd need to overcome to convert this more complete example?

Reply:
I'll try this out. I think it should be straight-forward as it's just base color and specular roughness + normlmap which is mapped.